tests/unit/fixtures/single-request.js

```diff
@@ -36,8 +36,23 @@ module.exports = {
         description: "This endpoint is used to get the `access_token`\n\nIt requires the following secret client credentials to be sent as part of the form body along with the `authentication code` obtained as part of the `redirect_uri` from the previous request.\n\n> code: xWnkliVQJURqB2x1\n>\n> grant_type: authorization_code\n>\n> redirect_uri: https://www.getpostman.com/oauth2/callback\n>\n> client_id: abc123\n>\n> client_secret: ssh-secret\n\nIf the correct credentials are not passed, the server returns with a `401 Unauthorized` response.",
         headers: "A:B\nC:D\n// E: F",
         method: "POST",
-        pathVariables: {},
-        url: "https://yo.postman.wtf/oauth2/token",
+        pathVariables: {
```

Comment:
Like we've said a number of times, please don't change existing fixtures, just add new ones if necessary.